spark-dependencies/pom.xml

```diff
@@ -480,7 +482,7 @@
     <profile>
       <id>cassandra-spark-1.5</id>
       <properties>
-        <spark.version>1.5.1</spark.version>
+        <spark.version>1.5.2</spark.version>
```

Comment:
Does R interpreter also require changing default spark version or should that be in separate PR?

In my experience, review\testing and eventually merging will be *much* faster if there are less changes per PR.